dnf/base.py

```diff
@@ -1064,6 +1066,15 @@ def download_packages(self, pkglist, progress=None, callback_total=None):
             percent = 100 - real / full * 100
             logger.info(msg, full / 1024 ** 2, real / 1024 ** 2, percent)
 
+        if self.conf.destdir:
+            dnf.util.ensure_dir(self.conf.destdir)
+            for pload in payloads:
+                shutil.copy(pload.pkg.repo.pkgdir + "/" + os.path.basename(pload.pkg.location), self.conf.destdir)
+                if self.conf.destdir.endswith('/'):
```

Comment:
if can be replaced by os.chmod(os.path.join(self.conf.destdir, os.path.basename(pload.pkg.location)), stat.S_IRWXO).